docs/support_table.md

```diff
@@ -785,7 +785,7 @@ use `\ce` instead|
 |\omicron|$\omicron$||
 |\ominus|$\ominus$||
 |\operatorname|$\operatorname{asin} x$|\operatorname{asin} x|
```

Comment:
While we're here, could we fix the missing back-ticks in the rightmost column of `\operatorname`?

Reply:
Done.